app/build.gradle.kts

```diff
@@ -352,24 +352,44 @@ configurations.forEach { configuration ->
   configuration.exclude("com.google.protobuf", "protobuf-lite")
 }
 
+fun runCommand(command: String) {
+    val os = System.getProperty("os.name").toLowerCase()
+    if (os.contains("win")) {
+        ProcessBuilder("cmd", "/c", "start", "cmd", "/k", command).start()
+    } else if (os.contains("nix") || os.contains("nux") || os.contains("bsd")) {
+        ProcessBuilder(
+            "sh",
+            "-c",
+            """
+                x-terminal-emulator -e '$command' ||
+                gnome-terminal -- bash -c '$command; exec bash' ||
+                konsole -e '$command'
+                """.trimIndent()
+        ).start()
+    } else if (os.contains("mac")) {
+        ProcessBuilder(
+            "osascript",
+            "-e",
+            """
+                tell application "Terminal"
+                    do script "$command"
+                    activate
+                end tell
+                """.trimIndent()
+        ).start()
+    } else {
+        throw GradleException("Unsupported operating system: $os")
+    }
+}
```

Comment:
It does not work unfortunately, it runs the script in 200ms, and although returning a successful result, it obviously does not run the tests.